charts/backingservices/values.yaml

```diff
@@ -47,6 +55,24 @@ elasticsearch:
   # Shrink default JVM heap.
   esJavaOpts: "-Xmx1024m -Xms1024m"
   # Allocate smaller chunks of memory per pod.
+  protocol: http
+  # This section specifies the elasticsearch cluster configuration for authentication.
+  esConfig:
+    elasticsearch.yml: |
+      xpack.security.enabled: true
+      xpack.security.transport.ssl.enabled: true
+  #Extra Environmental Variables for elasticsearch basic authentication , Values are read from Secrets
```

Comment:
Use this section to include additional, supported environmental variables for Elasticsearch basic authentication. The parameter values can be read from a specified secrets file.